### PR TITLE
Reader Sidemenu responsive 

### PR DESCRIPTION
**Pull request by nolawi**
Connects #4802

### Changes
- [ ] creates two breakpoints 920px & 1240px 
- [ ] Sets min and max width 250px & 380px
- [ ] Sidebar width is 250px from 800px-920px and gradually sizes up to 380px
- [ ] Keeps all functionality
- [ ] Make PDF toolbar fit on smaller screen (860px for now)

### CODE REVIEW

File: client/app/reader/PdfSidebar.jsx [hunk 1]
```diff
@@ -143,11 +144,21 @@ export class PdfSidebar extends React.Component {
       </div>;
     });
 
+    let sideBarSmall = '250px';
```
Comment (mdbenjam) [inline client/app/reader/PdfSidebar.jsx]: I'd move these constants to the top of the file and make them `const` instead of `let`

File: app/assets/stylesheets/_reviewer.scss [hunk 2]
```diff
@@ -70,6 +70,7 @@ $caseflow-text-select: transparentize($caseflow-text-select-without-transparency
 
   &.hidden-sidebar {
     right: 0;
+    width: 100%;
```
Comment (mdbenjam) [inline app/assets/stylesheets/_reviewer.scss]: Why is this needed? Seems like it should be 0 if it's going to be anything.

File: client/app/reader/PdfSidebar.jsx [hunk 3]
```diff
@@ -143,11 +144,21 @@ export class PdfSidebar extends React.Component {
...
     const sidebarClass = classNames(
       'cf-sidebar-wrapper',
       { 'hidden-sidebar': this.props.hidePdfSidebar });
 
-    return <div className={sidebarClass}>
+    const sidebarWrapper = css({
+      width: '28%',
+      minWidth: sideBarSmall,
+      maxWidth: sideBarLarge,
+      '@media(max-width: 920px)': { width: sideBarSmall },
```
Comment (mdbenjam) [inline client/app/reader/PdfSidebar.jsx]: I think the ticket specifies stopping to get smaller at 800px. Is there a reason we used 920 here instead?

File: client/app/reader/PdfUI.jsx [hunk 4]
```diff
@@ -167,9 +168,30 @@ export class PdfUI extends React.Component {
...
+    });
+
+    const pdfToolbar = css({
+      '&&': { '@media(max-width: 1240px)': { width: 'max-content' }
+      }
+    });
+
+    const pdfToolbarRight = css({
+      '&&': { '@media(max-width: 1240px)': {
+        float: 'right'
```
Comment (mdbenjam) [inline client/app/reader/PdfUI.jsx]: What is this supposed to do? I don't see a difference from this property.

**Comment by gnakm** [comment]
@nolawi hey it looks like the category tag and document title are becoming aligned at a smaller screen. Could we change it so that it is center-aligned at all times?

File: client/app/reader/PdfSidebar.jsx (hunk 3 above)
Comment (nolawi) [reply client/app/reader/PdfSidebar.jsx]: this covers 800px-920px, 920 marks the point where sidebar hits its min-width, can be adjusted but this is where toolbar spacing issue come into play. LMK

File: client/app/reader/PdfUI.jsx (hunk 4 above)
Comment (nolawi) [reply client/app/reader/PdfUI.jsx]: Essentially, the grid is 1/3 for each section and floated left. On smaller screens I am collapsing the width of each column to be the width of its content, hence giving more space for the third column and then floating it to the right.
screen shot 2018-03-20 at 2 15 07 pm

**Comment by nolawi** [comment]
@gnakm I started on another PR switching to a flexbox which will resolve centering.
